jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/WebDriverFactory.java

```diff
@@ -7,12 +7,10 @@
 import com.magenic.jmaqs.selenium.constants.BrowserType;
 import com.magenic.jmaqs.selenium.constants.OperatingSystem;
 import com.magenic.jmaqs.selenium.constants.RemoteBrowserType;
-import com.magenic.jmaqs.selenium.constants.WebDriverFile;
 import com.magenic.jmaqs.selenium.exceptions.DriverNotFoundException;
 import com.magenic.jmaqs.selenium.exceptions.WebDriverFactoryException;
-import com.magenic.jmaqs.utilities.helper.Config;
-import com.magenic.jmaqs.utilities.helper.ConfigSection;
 import com.magenic.jmaqs.utilities.helper.StringProcessor;
+import io.github.bonigarcia.wdm.WebDriverManager;
```

Comment:
We should have a fall back in case WebDriverManager functionality is blocked.
This could easily be put in a different PR.